app/models/metric/rollup.rb

```diff
@@ -152,6 +155,14 @@ def self.rollup_hourly(obj, hour, _interval_name, _time_profile, new_perf, orig_
     new_perf.reverse_merge!(orig_perf)
     new_perf.merge!(Metric::Processing.process_derived_columns(obj, new_perf, hour)) unless DERIVED_COLS_EXCLUDED_CLASSES.include?(obj.class.base_class.name)
 
+    # Add metric data collected from vmdb
+    if orig_perf[:resource_type] == "ContainerNode"
```

Comment:
@simon3z 
> for sure this check doesn't look nice here (code-wise)

Yes, it will move into process_derived_columns, the name of the column will change to derived_?createcontainer?_?rate?
> why the ContainerGroup count is attached to each ContainerNode?

I do not know who has the "createcontainer_rate" property, my best guess was ContainerNode. Help will be appreciated about this :-)